File: Samples/Verbum/displaying-bookmark-names-in-the-bookmarks-pane-of-pdf-viewer-for-a-document-exported-to-p-t420469-16.1.4-/VB/BookmarkTitleUpdater/Program.vb
Imports System
Imports System.Collections.Generic
Imports System.Windows.Forms
Imports DevExpress.LookAndFeel

Namespace BookmarkTitleUpdater
    Friend NotInheritable Class Program

        Private Sub New()
        End Sub

        ''' <summary>
        ''' The main entry point for the application.
        ''' </summary>
        <STAThread> _
        Shared Sub Main()
            Application.EnableVisualStyles()
            Application.SetCompatibleTextRenderingDefault(False)

            DevExpress.Skins.SkinManager.EnableFormSkins()
            DevExpress.UserSkins.BonusSkins.Register()
            UserLookAndFeel.Default.SetSkinStyle("Office 2016 Colorful")

            Application.Run(New Form1())
        End Sub
    End Class
End Namespace

File: Samples/Verbum/displaying-bookmark-names-in-the-bookmarks-pane-of-pdf-viewer-for-a-document-exported-to-p-t420469-16.1.4-/CS/BookmarkTitleUpdater/Program.cs
using System;
using System.Collections.Generic;
using System.Windows.Forms;
using DevExpress.LookAndFeel;

namespace BookmarkTitleUpdater
{
    static class Program
    {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        static void Main()
        {
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);

            DevExpress.Skins.SkinManager.EnableFormSkins();
            DevExpress.UserSkins.BonusSkins.Register();
            UserLookAndFeel.Default.SetSkinStyle("Office 2016 Colorful");

            Application.Run(new Form1());
        }
    }
}